thorlcr/thorutil/thmem.hpp

```diff
@@ -88,7 +88,7 @@ class OwnedConstThorRow
     inline OwnedConstThorRow(const void * _ptr)             { TESTROW(_ptr); ptr = _ptr; }
     inline OwnedConstThorRow(const OwnedConstThorRow & other)   { ptr = other.getLink(); }
 
-    inline ~OwnedConstThorRow()                             { ReleaseThorRow(ptr); }
+    inline virtual ~OwnedConstThorRow()                     { ReleaseThorRow(ptr); }
```

Comment:
I am not sure if this is a good idea.. may be ok, but consequence I think is that if you allocate a large number of OwnedConstThorRow objects now, you have a VMT for each where didn't before. So doubles the space required.

@ghalliday - can you comment please?

Not strictly needed as not overloaded by CThorRowHistory.
It may be better if you leave OwnedConstThorRow untouched and contain it as a member of CThorRowHistory